app/controllers/chats_controller.rb

```diff
@@ -0,0 +1,73 @@
+class ChatsController < ApplicationController
+  def index
+    @chats = Current.user.chats.order(created_at: :desc)
+  end
+
+  def new
+    @chat = Current.user.chats.new
+  end
+
+  def create
+    @chat = Current.user.chats.create
+    @message = @chat.messages.create(
+      user: Current.user,
+      content: chat_params[:content],
+      role: "user"
+    )
+
+    # Stub reply from bot
+    reply = @chat.messages.create(
```

Comment:
This could be moved to the model to clean things up a bit. Something like `Message.create_pending_assistant_message`